tools/lib/makestuff.js

```diff
@@ -26,10 +26,10 @@ code.hljs {
 }
 `.trim();
 
-function installCleanCSS(file, dest) {
+function installCleanCSS(file, dest, minify) {
   const theme = fs.readFileSync(file, { encoding: "utf8" });
   const content = DEFAULT_CSS + "\n" + theme;
```

Comment:
```suggestion
function installCleanCSS(file, dest, opts = {}) {
  const minify = opts.minify ?? true; // default is to minify
  
  const theme = fs.readFileSync(file, { encoding: "utf8" });
  const content = DEFAULT_CSS + "\n" + theme;
```

Reply:
Done, please enable the CI to double check the CI is passing.